store/src/java/com/zimbra/cs/account/soap/SoapDistributionList.java

```diff
@@ -81,11 +81,20 @@
                 SoapProvisioning.getAttrs(e), prov);
         addDlm(e, getRawAttrs());
     }
-
+    
+    //ZBUG-651: This bug is about to get correct member details for gadl command;
```

Comment:
add the comments to method documentation here and below

Reply:
Method documentation is added.